modules/server/src/main/ts/bedrock/cli/Clis.ts

```diff
@@ -58,7 +58,8 @@ export const forAuto = (directories: Directories, argv: string[] = process.argv)
     ClOptions.devicefarmRegion,
     ClOptions.browserVersion,
     ClOptions.platformName,
-    ClOptions.useSelenium
+    ClOptions.useSelenium,
+    ClOptions.clipboardPermission
```

Comment:
This doesn't seem to be actually used anywhere? It always adds `--enable-clipboard` on chrome

Reply:
Yeah.. sorry should've been a draft pr, changing that now. 😬